notes_project/notes_project/urls.py

```diff
@@ -15,16 +15,14 @@
 """
 from django.contrib import admin
 from django.urls import path
-from django.conf.urls import url
-from django.conf.urls import include
+from django.conf.urls import url, include
 from notes_app import views
 from notes_app.views import ClassRoomListView, ClassRoomDetailView, UserLogin
 
 urlpatterns = [
     path("admin/", admin.site.urls),
     url(r"^$", views.index, name="index"),
     url(r"^", include("notes_app.urls")),
-    # url(r'^user_login/$', views.user_login, name='user_login'),
```

Comment:
You deleted this because we change this function in based view class?

Reply:
code refactoring